adapters/pancakeswap/src/sdk/lib.ts

```diff
@@ -164,6 +165,24 @@ export const getV3UserPositionsAtBlock = async (
   return [...resultMap.values()];
 };
 
+export const getSickles = async () => {
+  const query = `query fetchSickles {
```

Comment:
you may need to also provide block number in subgraph
`block: { number: ${blockNumber} },`

Reply:
Let's say we pass a block number, and the sickle contract Deployed event is before this block, but it has a balance. We will miss the owner in this case. This is why i decided to kerp the function without a block number. Correct me if i am wrong